image-patching/stopgap-script/nuage_patching_config.yaml

```diff
@@ -6,9 +6,12 @@ ImageName: "overcloud-full.qcow2"
 # DeploymentType for your Deployment
 # type: string
 # required ( Please choose one of them )
-# Eg:  ["ovrs"] or ["avrs"]
+# Eg:  ["ovrs"] --> OVRS deployment
+#      ["ovrs"] --> A/VRS deployment
```

Comment:
typo, this should be "avrs" and say (VRS + AVRS)

Reply:
Done